package-lock.json

```diff
@@ -5090,6 +5090,17 @@
             "unique-filename": "^1.1.1"
           }
         },
+        "chalk": {
```

Comment:
Are you meaning to add these?

Reply:
No clue, this was updated when I ran `npm install` I think 🤔